test/unittests/commands/xmlupload/make_rdf_graph/test_make_values.py

```diff
@@ -274,14 +219,36 @@ def test_richtext(self, lookups: Lookups) -> None:
         permissions = next(result.objects(val_bn, KNORA_API.hasPermissions))
         assert permissions == PERMISSION_LITERAL
 
-    def test_date(self, lookups: Lookups) -> None:
+    def test_richtext_with_reference(self, lookups: IRILookup) -> None:
+        res_bn = BNode()
+        text = 'Comment with <a class="salsah-link" href="IRI:res_one:IRI">link to res_one'
+        prop = IntermediaryRichtext(
+            FormattedTextValue(text), onto_str("hasRichtext"), None, None, resource_references=set("res_one")
+        )
+        result, prop_name = _make_one_prop_graph(prop, res_bn, lookups)
+        assert len(result) == 4
+        assert prop_name == ONTO.hasRichtext
+        val_bn = next(result.objects(res_bn, prop_name))
+        rdf_type = next(result.objects(val_bn, RDF.type))
+        assert rdf_type == KNORA_API.TextValue
+        value = next(result.objects(val_bn, KNORA_API.textValueAsXml))
+        expected_text = (
+            '<?xml version="1.0" encoding="UTF-8"?>\n'
```

Comment:
where does the newline come from? not that it matters, just curiosity

Reply:
it comes from copy pasting... thanks for noticing it.